Algorithms.Test/ShuffleTheArrayTest.cs

```diff
@@ -0,0 +1,69 @@
+            int elements        = 420;
```

Comment:
As you can see, all your tests are the happy path. 
What happens if I sent `elements = -1`, what happens if I sent `nums Array = new int[0]` and then `elements = 10`... etc etc. That's the meaning of edge cases

Reply:
Got it. I'll change my unit tests